.env

```diff
@@ -0,0 +1 @@
+VITE_IS_PROD=0 
```

Comment:
so this file shouldn't actually be pushed into the repo oof
a `.env` file is meant to store environment variables that we usually want to keep hidden from the public eye.
you'll need to create a .gitignore file, I'll send this in slack!